backends/rapidpro/status.go

```diff
@@ -265,11 +265,11 @@ UPDATE msgs_msg SET
 		END,
 	sent_on = CASE 
 		WHEN 
-			s.status = 'W' 
+			s.status IN ('W', 'S', 'D') AND sent_on IS NULL
```

Comment:
what about the msgs with R? we clear the sent on on them?

Reply:
we don't use `R` any more - resending requeues messages as QUEUED

Reply:
never mind we use pending on them